huaweicloud/services/cce/resource_huaweicloud_cce_node.go

```diff
@@ -1247,8 +1247,7 @@ func getResourceIDFromJob(ctx context.Context, client *golangsdk.ServiceClient,
 		}
 	}
 
-	return "", fmt.Errorf("error fetching %s Job resource id", subJobType)
-
+	return "", fmt.Errorf("error fetching the resource ID of the specified job (type: %s)", subJobType)
```

Comment:
use `in` or `from` instead of `of` will be better

Reply:
`from` is better yep.